app/middlewares/errors.js

```diff
@@ -10,7 +10,8 @@ const statusCodes = {
   [errors.CREDENTIALS_ERROR]: 401,
   [errors.AUTHORIZATION_ERROR]: 401,
   [errors.TOKEN_EXPIRATION_ERROR]: 401,
-  [errors.FORBIDDEN_ERROR]: 403
+  [errors.FORBIDDEN_ERROR]: 403,
+  [errors.EXTERNAL_API_ERROR]: 500
```

Comment:
An external error, the status code is 503.
Check  [status code page](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)